src/OpenTelemetry/CHANGELOG.md

```diff
@@ -16,8 +16,13 @@
   ([#4375](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4375))
 
 * Tweaked the behavior of the `OpenTelemetryLoggerOptions.ParseStateValues`
-  flag, obsoleted `LogRecord.State` and `LogRecord.StateValues` properties, and
-  added `LogRecord.Body` and `LogRecord.Attributes` properties.
+  flag (attributes are now automatically included for most log messages by
```

Comment:
There is also the change that from now on `State` could be `null` even when `ParseStateValues` is set to `false`. (when state is `IReadOnlyList<KeyValuePair<string, object?>>` or `IEnumerable<KeyValuePair<string, object?>>`)

I think we should mention this change as well.

Reply:
I just pushed a new version. LMK what you think.